app/themes/default/components/ui/admin/ProductCategoryList.tsx

```diff
@@ -1,12 +1,195 @@
-import { useContext } from 'react'
+import { useFetcher, useSubmit } from '@remix-run/react'
+import { useContext, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import AdminContext from '~/contexts/adminContext'
+import { Button } from '../button'
+import { Dialog, DialogContent, DialogFooter, DialogHeader, DialogTitle } from '../dialog'
+import { Input } from '../input'
+import { Label } from '../label'
+import { Spinner } from '../spinner'
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '../table'
 
 const ProductCategoryList = () => {
   const { t } = useTranslation()
-  const { storeSettings } = useContext(AdminContext)
+  const { categoryItem } = useContext(AdminContext)
```

Comment:
I think **CategoryItem** should NOT be stored in AdminContext, coz it's only relevant to the ProductCategoryList component. The objects stored in the AdminContext are shared by **all** the Admin pages.

I'd suggest to pass the CategoryItem as a prop of the ProductCategoryList, such as ```<ProductCategoryList productItem={<the categoryItem object or array>} />```